### PR TITLE
C++: Tidy up old QL headers

### PR DESCRIPTION
**Pull request by geoffw0**
Tidy up some strange headers on old QL files.  See https://github.com/github/semmle-jira-migration/issues/294.

### CODE REVIEW

File: cpp/ql/src/Architecture/General Namespace-Level Information/GlobalNamespaceClasses.ql [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /**
  * @name Global namespace classes
  * @description Finds classes that belong to no namespace.
- * @kind table
+ * @kind problem
+ * @problem.severity recommendation
```
Comment (MathiasVP) [inline cpp/ql/src/Architecture/General Namespace-Level Information/GlobalNamespaceClasses.ql:5]: Do we want a `@precision` tag as well?

**Review by MathiasVP — COMMENTED** [review]
LGTM otherwise!

File: cpp/ql/src/Architecture/General Namespace-Level Information/GlobalNamespaceClasses.ql (hunk 1 above)
Comment (geoffw0) [reply cpp/ql/src/Architecture/General Namespace-Level Information/GlobalNamespaceClasses.ql:5]: I'm a bit reluctant as I believe it's precise, but I don't want it picked up to run somewhere (like LGTM) as this is a somewhat specialized query.

Comment (MathiasVP) [reply cpp/ql/src/Architecture/General Namespace-Level Information/GlobalNamespaceClasses.ql:5]: That's fair. I see now that we have plenty of other `@problem.severity recommendation` queries without a `@precision` tag so it's probably fine to introduce another one.

**Review by MathiasVP — APPROVED** [review]

**Pull request merged**